SU-HSDO/suhumsci
# Additional Global H&S Theme Fixups

# READY FOR REVIEW

# Summary
- Reduce font weight of title in vertical card patterns
- Address text hovering issue inside of Decanter buttons and re-import Decanter Overrides
- Increase body width for three column with image layout
- Reduce font size for third and fourth level navigation in subnav

# Needed By (Date)
- End of sprint (10/25)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Verify that the above is propagating; I can provide links for testing as needed (BUGHERD tickets for each of these items)

# Affected Projects or Products
- France-Stanford
- Archaeology
- suhumsci (theme)

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/HSD8-471
- https://stanfordits.atlassian.net/browse/HSD8-472
- https://stanfordits.atlassian.net/browse/HSD8-473
- https://stanfordits.atlassian.net/browse/HSD8-474

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
